### DISCUSSION TITLE
How to add your callsign to a suggestion

### DISCUSSION THREAD

**Discussion by wdlier** [wavelog/wavelog#649] [Q&A] [answered]
![1](https://github.com/user-attachments/assets/4e374607-6b9e-49e4-8fa1-e2f95bd09fbd)
I've been searching for a long time and can't seem to find an answer. Does anyone know? Thank you very much.

**Comment by HB9HIL** [comment, marked as answer]
The SCP files are databases of active contesters. SCP stands for "Super Check Partial" which means you can search for partial callsigns and you can find possible matches. 

Currently there is the ClublogSCP file used in Wavelog. We plan to add a second one in #570 .

In general these SCP files are generated from contest logs. So if you want your callsign to be added here to have to work contests. For example clublog generates a SCP file from callsigns with **more then 40 QSOs in the clublog database over the last 3 years** ([info here](https://clublog.freshdesk.com/support/solutions/articles/54907-super-check-partial-files-scp-)). Other providers like [https://www.supercheckpartial.com/](https://www.supercheckpartial.com/) use contest logs from common big contests (ARRL Contests or the russian dx contest just to name two of them).

**Reply by wdlier** [reply]
Thank you very much for your response.
